wrappers/openvino/CMakeLists.txt

```diff
@@ -10,6 +10,12 @@ if( NOT DEFINED INTEL_OPENVINO_DIR  OR  NOT IS_DIRECTORY ${INTEL_OPENVINO_DIR} )
 	message( FATAL_ERROR "Invalid OpenVINO directory specified with INTEL_OPENVINO_DIR" )
 endif()
 
+if (NOT BUILD_WITH_CPU_EXTENSIONS)
+    message( STATUS "Disable CPU extensions for OpenVINO" )
```

Comment:
Disabling.
Doesn't the message come out wrong:
`STATUSDisable ...`
?

Reply:
No, 
![image](https://user-images.githubusercontent.com/64067618/116405069-eb57ce80-a837-11eb-8413-8f72dfa375a9.png)